src/components/Menus/Sidebar/Sidebar.jsx

```diff
@@ -43,6 +45,13 @@ const Sidebar = ({ options = [] }) => {
       position: "top",
       panel: ImagePanel,
     },
+    {
+      name: "download",
+      icon: DownloadIcon,
+      title: t("sidebar.information"),
```

Comment:
You need to add some copy into the `i18n.js` for the `title` on the sidebar icon

Reply:
Thanks for spotting this - done!